br/pkg/task/backup_raw.go

```diff
@@ -49,6 +51,10 @@ func DefineRawBackupFlags(command *cobra.Command) {
 	command.Flags().Bool(flagRemoveSchedulers, false,
 		"disable the balance, shuffle and region-merge schedulers in PD to speed up backup.")
 
+	command.Flags().Duration(flagSafeInterval, utils.DefaultBRSafeInterval,
```

Comment:
Suggest to hide safe interval. It's difficult for common users to set this argument.

Reply:
done